src/main/java/Deadline.java

```diff
@@ -0,0 +1,26 @@
+import javax.sound.midi.SysexMessage;
+
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by){
+        super(description);
+        this.by = by;
+    }
+
+
+    public String getStatusIcon() {
+        return "[D]" + "[" + super.getStatusIcon() +"]";
+    }
+
+
+    @Override
+    public String getDescription() {
+        return super.getDescription() + " " + "(" + by + ")";
+    }
+
```

Comment:
These  lines do not serve a special purpose, and you could consider to remove them in favour of [code readability](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#delete-dead-code).